kieserver/modules/kieserver/module.yaml

```diff
@@ -23,22 +24,22 @@ envs:
 - name: "BUSINESS_CENTRAL_DISTRIBUTION_EAP"
   value: "jboss-eap-7.4"
 - name: "JBPM_WB_KIE_SERVER_BACKEND_JAR"
-  value: "jbpm-wb-kie-server-backend-7.67.2.Final-redhat-00004.jar"
+  value: "jbpm-wb-kie-server-backend-7.67.2.Final-redhat-00005.jar"
 # remember to also update "JBPM_WB_KIE_SERVER_BACKEND_JAR" value
 artifacts:
 - name: "bamoe_kie_server_distribution.zip"
   # bamoe-8.0.3-kie-server-ee8.zip
-  md5: "77d9412aca4dc588bc2d5685c64bfc16"
+  md5: "d9d34421617e6585bb9bef46b166bce8"
 - name: "bamoe_business_central_distribution.zip"
   # bamoe-8.0.3-business-central-eap7-deployable.zip
-  md5: "9f581657bd89170122c3924c4dd92f22"
+  md5: "371381a425854b2f5ca840e10b3ca745"
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
```

Comment:
@spolti @bacciotti  for the RHPAM 7.13.3 the slf4j-simple.jar was updated from version `1.7.22.redhat-2` to `1.7.30.redhat-00005`, see PR https://github.com/jboss-container-images/rhpam-7-image/pull/643

I think we should do the same for BAMOE 8.0.3. WDYT?

Reply:
I think we should update for BAMOE 8.0.3 as well. Good point.